src/libstd/io/buffered.rs

```diff
@@ -752,17 +752,31 @@ impl<W> fmt::Display for IntoInnerError<W> {
 ///     let file = File::create("poem.txt")?;
 ///     let mut file = LineWriter::new(file);
 ///
-///     for &byte in road_not_taken.iter() {
-///        file.write(&[byte]).unwrap();
-///     }
+///     file.write_all(b"I shall be telling this with a sigh")?;
+///
+///     // No bytes are written until a newline is encountered (or
+///     // the internal buffer is filled).
+///     assert_eq!(fs::read_to_string("poem.txt")?.as_bytes(), b"");
```

Comment:
I think `fs::read_to_string(...)?.as_bytes()` can just be `fs::read(...)?`

https://doc.rust-lang.org/nightly/std/fs/fn.read.html

Reply:
Alternatively this can be `assert_eq!(fs::read_to_string("poem.txt")?, "")`.

Reply:
Fixed.